app/models/account.rb

```diff
@@ -0,0 +1,6 @@
+class Account < ApplicationRecord
+  belongs_to :user
+
+  has_many :creditor_user_cues, class_name: "UserCue", inverse_of: :creditor_account
```

Comment:
@bibiferreira :creditor_user_cues?

Reply:
this is just an extra, not really mandatory but in this way we could list all the cues associated to an account

Reply:
Ok gotcha, nice